packages/react/src/autocomplete-context/AutocompleteContext.ts

```diff
@@ -0,0 +1,13 @@
+import type { UseComboboxReturnValue } from "downshift";
+
+import { createContext } from "@radix-ui/react-context";
+
+export const [AutocompleteContextProvider, useAutocompleteContext] =
+  createContext<{
+    disabled?: boolean;
```

Comment:
we shouldn't have this here - disabled should be set on the input directly if the intent is to disable the input